spec/adaptors/quickbooks_online/operations/invoice/create_spec.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+support :input_helpers,
+        :operation_shared_examples,
+        :quickbooks_online_helpers
+
+RSpec.describe LedgerSync::Adaptors::QuickBooksOnline::Invoice::Operations::Create do
+  include InputHelpers
+  include QuickBooksOnlineHelpers
+
+  let(:customer) do
+    LedgerSync::Customer.new(customer_resource(ledger_id: '123'))
```

Comment:
Please use `FactoryBot` so that we ensure variance does not matter.  And then we can get rid of the resource hashes.

Reply:
Tbh for anything that adds `ledger_id` we dont need any factories. We need that object only as a `Ref`